src/components/menu/menu.tsx

```diff
@@ -36,7 +36,7 @@ export class CalciteNavMenu {
   @Prop() label: string;
 
   // todo evaluate slotted content and determine if it is a nav menu item, then limit # rendered when auto-collapsing based on width of parent
-  @Prop({ mutable: true }) minCollapsedItems?;
+  @Prop() minCollapsedItems: boolean;
```

Comment:
We can remove this, for the time being, until we add back the auto-collapse functionality?

Reply:
sure, removed.